namespaces/live-1.cloud-platform.service.justice.gov.uk/check-financial-eligibility-uat/resources/main.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  backend "s3" {}
+}
+
+provider "aws" {
+  region = "eu-west-1"
```

Comment:
you probably want eu-west-2?

Reply:
the apply-for-legal-aid application is hosted on ```eu-west-1```. this application will mainly be used by apply, hence choosing ```west-1```, but if it is preferred for live-1 to be on ```west-2``` happy to change it.